### PR TITLE
Horizontally scrollable tabs

### PR DESCRIPTION
**Pull request by krauselukas**
Peek 2020-03-05 17-37
Peek 2020-03-05 17-42

### CODE REVIEW

**Comment by krauselukas** [comment]
The PR is WIP, since the scrollable tabs are only implemented for one of the view's yet.

**Comment by krauselukas** [comment]
[Main Tabs](https://trello.com/c/U2RHQ0nr/1335-main-tabs)

**Comment by hennevogel** [comment]
You can't DRY all those `@namespace` assignments somehow?

**Comment by krauselukas** [comment]
> You can't DRY all those `@namespace` assignments somehow?

I can dry it up more, looking into that :)

File: src/api/app/assets/javascripts/webui/responsive_ux.js [hunk 1]
```diff
@@ -22,3 +22,15 @@ $(function () {
     $('#username').focus();
   });
 });
+
+/* bootstrap's tabs javascript doesn't remove the active class
+   when tabs are implemented without the usage of ul and li */
```
Comment (hennevogel) [inline src/api/app/assets/javascripts/webui/responsive_ux.js]: Have you considered to be a good open source citizen and open an issue over at bootstrap? 😄

**Pull request merged**